Plugin/src/main/java/xyz/kyngs/librelogin/common/config/MessageKeys.java

```diff
@@ -145,6 +145,14 @@ public class MessageKeys {
             ConfigurateHelper::getString
     );
 
+
+    public static final ConfigurationKey<String> KICK_TOO_MANY_LOGIN_TRIES = new ConfigurationKey<>(
+            "kick-too-many-login-tries",
+            "Suspicious activity! Too many login tries!",
+            "This message is displayed when the player is kicked because of trying to login many times!",
+            ConfigurateHelper::getString
+    );
+
```

Comment:
Wouldn't it make more sense to reuse the KICK_ERROR_PASSWORD_WRONG here?

Reply:
Yup, I didn't realize there was such a message.